lib/matplotlib/mathtext.py

```diff
@@ -830,16 +856,20 @@ def _get_glyph(self, fontname, font_class, sym, fontsize, math=True):
 
         if not found_symbol:
             if self.cm_fallback:
-                if isinstance(self.cm_fallback, BakomaFonts):
-                    _log.warning(
-                        "Substituting with a symbol from Computer Modern.")
-                if (fontname in ('it', 'regular') and
-                        isinstance(self.cm_fallback, StixFonts)):
-                    return self.cm_fallback._get_glyph(
-                            'rm', font_class, sym, fontsize)
-                else:
-                    return self.cm_fallback._get_glyph(
-                        fontname, font_class, sym, fontsize)
+                if type(self.cm_fallback) is BakomaFonts:
```

Comment:
Perhaps something like `if self.cm_fallback is not None: _log.warning("Substituting with a symbol from %s", font.family)` would be more general (where font is suitably defined so that font.family works.).

Reply:
I don't get it, this would need to add some self.font_name to the base TrueTypeFonts class and assign it for each of the Font classes to check for it here. Unsure if that makes sense.

Reply:
No, rather I mean you're going to call findfont() to get a real font object anyways(?) in that case, and that font object has a family attribute.